lib/rdf/vocab/acl.rb

```diff
@@ -104,9 +110,9 @@ class ACL < RDF::StrictVocabulary("http://www.w3.org/ns/auth/acl#")
       range: "rdfs:Class".freeze,
       type: "rdf:Property".freeze
     property :owner,
-      comment: %(The person or other agent which owns this. 
+      comment: %(The person or other agent which owns this.
     For example, the owner of a file in a filesystem.
-    There is a sense of right to control.   Typically defaults to the agent who created
+    There is a sense of right to control.   Typically defaults to the agent who craeted
```

Comment:
did the upstream vocab introduce a misspelling of "created", or did you do that? :D